File: lib/Music.dart
class Music {
  final String _id;
  String get id => _id;
  final String _title;
  String get title => _title;
  final String _artist;
  String get artist => _artist;
  final String _album;
  String get album => _album;
  final int _duration;
  int get duration => _duration;
  final String _artUri;
  String get artUri => _artUri;

  
  Music (
    this._id,
    this._title,
    this._artist,
    this._album,
    this._duration,
    this._artUri
  );
}

File: android/app/src/main/java/com/example/riot_quiche/Music.java
package com.example.riot_quiche;


public class Music {
    private String id;
    private String title;
    private String artist;
    private String album;
    private Long duration;
    private String artUri;


    public Music () {}

    public Music (
            String id,
            String title,
            String artist,
            String album,
            Long duration,
            String artUri) {

        this.id = id;
        this.title = title;
        this.album = album;
        this.artist = artist;
        this.duration = duration;
        this.artUri = artUri;
    }

    public String getId () {
        return id;
    }
    public String getTitle () {
        return title;
    }
    public String getAlbum () {
        return album;
    }
    public String getArtist () {
        return artist;
    }
    public Long getDuration () {
        return duration;
    }
    public String getArtUri () {
        return artUri;
    }
}
